src/components/DraggableItem.jsx

```diff
@@ -1,43 +1,52 @@
 /* eslint-disable react/jsx-props-no-spreading */
-import React from 'react';
-import { Draggable } from 'react-beautiful-dnd';
-import { Box, Text } from '@chakra-ui/react';
+import React, { useState } from 'react';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'useState' is defined but never used.